packages/database/src/models/SyncSession.ts

```diff
@@ -50,7 +52,7 @@ export class SyncSession extends Model {
     await this.sequelize.query(
       `
       UPDATE "sync_sessions"
-      SET "debug_info" = (COALESCE("debug_info"::jsonb, '{}'::jsonb) || $data::jsonb)::json
+      SET "parameters" = (COALESCE("parameters"::jsonb, '{}'::jsonb) || $data::jsonb)::json
```

Comment:
```suggestion
      SET "parameters" = (COALESCE("parameters", '{}'::jsonb) || $data::jsonb)
```

Might even get away with less typecasts